ruby-challenges.rb

```diff
@@ -10,24 +14,91 @@
 num3 = 221
 # Expected output: '221 is odd'
 
+stored_nums = num1, num2, num3
 
+def even_or_odd array
+    array.select do |value|
+        if value.odd?
+        puts "#{value.to_s} is odd."
+        else
+        puts "#{value.to_s} is even."
+        end
+    end
+end
```

Comment:
Make sure to pay attention to the prompt.  Your input is a number and not an array.  While you could definitely use this approach to determine if a value is odd or even in an array, this is more complex than the prompt is asking.  You only need to take a number as a param and run your conditional on that number.